dcs/settings.py

```diff
@@ -17,9 +17,6 @@
 # Quick-start development settings - unsuitable for production
 # See https://docs.djangoproject.com/en/1.7/howto/deployment/checklist/
 
-# SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = 'tik8g&=7)8mjbv@ji2r#=7i_yz0n_)_uu8%ad#0^vkks40%e5h'
```

Comment:
security reason. sebenarnya bukan di pindahkan ke local_settings.py tetapi di remove dari `settings.py`.

`__local_settings.py` cuma sebagai template, jadi itu bakalan di replace dengan aman. Kalo kita taru di `settings.py`, then people take it for granted. (not good)